tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -8,23 +8,34 @@
 """
 import pytest
 
-from ..models.utils import check_outputs_equal
+from ..models.utils import check_logprobs_close, check_outputs_equal
 
-MODELS = [
+EXACT_MATCH_MODELS = [
     "facebook/opt-125m",
     "meta-llama/Llama-2-7b-hf",
 ]
 
+E5M2_KV_MODELS = [  # type: ignore
+    # does not work with fp8 kv cache kernel
+    # - CUDA illegal memory access - undiagnosed
+    # "facebook/opt-125m",
+]
+E4M3_KV_MODELS = [
+    "nm-testing/Qwen2-1.5B-Instruct-FP8-K-V",
+    "nm-testing/TinyLlama-1.1B-compressed-tensors-kv-cache-scheme"
+]
+LOG_PROBS_MODELS = E4M3_KV_MODELS + E5M2_KV_MODELS
+
 
-@pytest.mark.parametrize("model", MODELS)
+@pytest.mark.parametrize("model", EXACT_MATCH_MODELS)
 @pytest.mark.parametrize("dtype", ["half"])
 @pytest.mark.parametrize("max_tokens", [32])
 @pytest.mark.parametrize("chunked_prefill_token_size", [1, 4, 16])
 @pytest.mark.parametrize("enforce_eager", [False, True])
 # NOTE: Increasing this in this suite will fail CI because we currently cannot
 # reset distributed env properly. Use a value > 1 just when you test.
 @pytest.mark.parametrize("tensor_parallel_size", [1])
-def test_models(
+def test_models_hf_exact_string_match(
```

Comment:
I now feel that we don't have to rename this test. Just leave it here to indicate that we are testing the most common use case (i.e., fp16).